### ISSUE TITLE
Script visual editor breaks multiple commands statements

### ISSUE THREAD

**Issue by skylarv** [home-assistant/frontend#9783]
### The problem

Using remote.send_command in a script and passing multiple commands via dashes on new lines works, however immediately after saving the script, clicking back into it loads the visual editor, and the visual editor then breaks it since it formats the commands with commas.

For example, good formatting via yaml editor and it sends the multiple commands successfully:
```
service: remote.send_command
data:
  delay_secs: 1
  command:
   - top_menu
   - home
target:
  entity_id: remote.master_bedroom_tv_4k
```

Clicking back into the script or switching to visual editor then hoses it and changes it to:
```
service: remote.send_command
data:
  delay_secs: 1
  command: top_menu,home
target:
  entity_id: remote.master_bedroom_tv_4k
```

### What is version of Home Assistant Core has the issue?

2021.8.6

### What type of installation are you running?

Home Assistant OS

### Example YAML snippet

```yaml
service: remote.send_command
data:
  delay_secs: 1
  command: top_menu,home
target:
  entity_id: remote.master_bedroom_tv_4k
```

**Comment by Friedi1970** [comment]
I think, this is exactly the same as the issue I already opened: https://github.com/home-assistant/frontend/issues/9753

**Comment by 49adrien** [comment]
Same problem for me : [#9542](https://github.com/home-assistant/frontend/issues/9542)

**Comment by loeffelpan** [comment]
Nothing changed, tested with HA 2021.11.5

**Comment by loeffelpan** [comment]
Seems now to be fixed with #10811 and HA 2021.12.x.
Can be closed.
